src/shogun/distance/AttenuatedEuclideanDistance.h

```diff
@@ -17,7 +17,9 @@
 #include <shogun/distance/RealDistance.h>
 #include <shogun/features/DenseFeatures.h>
 
-namespace shogun
+using namespace shogun;
```

Comment:
Here and everywhere when we do ```using namespace``` and then declare namespace, we don't do it nested. I think it should be like 
```namespace shogun
{
  namespace distance
  {```